SynRepoConfig/sync_files.csv

```diff
@@ -0,0 +1,2 @@
+SrcLoc, DestLoc
```

Comment:
Can you briefly explain to me what is this file for?

Reply:
This is currently a placeholder to copy files from SOFA to Caravel-SOFA-HD repo. 
like line 2 copies `FPGA1212_FLAT_HD_SKY_task` to `Caravel-SOFA-HD` with `OpenFPGA_task` name while deployment.